A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/A320_Neo_EICAS.js

```diff
@@ -71,14 +70,15 @@ class A320_Neo_EICAS extends Airliners.BaseEICAS {
         this.updateAnnunciations();
         this.updateScreenState();
         
-        var engineOn = Simplane.getEngineActive(0) || Simplane.getEngineActive(1);
-        var externalPower = SimVar.GetSimVarValue("EXTERNAL POWER ON", "bool");
-        var apuOn = SimVar.GetSimVarValue("L:APU_GEN_ONLINE", "bool");
-        let apuStatusChanged = (apuOn != this.APULastValue);
-        let externalPowerChanged = (externalPower != this.externalPowerLastValue);
-
-        var isACPowerAvailable = engineOn || apuOn || externalPower;
+        const engineOn = Simplane.getEngineActive(0) || Simplane.getEngineActive(1);
+        const externalPower = SimVar.GetSimVarValue("EXTERNAL POWER ON", "bool");
+        const apuOn = SimVar.GetSimVarValue("L:APU_GEN_ONLINE", "bool");
+        const isACPowerAvailable = engineOn || apuOn || externalPower;
```

Comment:
Is this going to change once your electricity changes are finished? If so could you start putting comments on lines like that saying something like "This will be replaced with proper electricity logic in the future"?

Reply:
These lines will get deleted once implementation is complete, while all the conditions that read from the simvars will still remain the same.